README.md

```diff
@@ -41,6 +41,22 @@ from anywhere in the repository.  This will run the tests and print out a short
 
 to find the specific ticket.
 
+### XML formatting, XML linting
+
+xmllint (http://xmlsoft.org) is used in GitHub action to check committed files format. The files can be reformatted running xmllint --format, such as:
+
+```bash
+for f in $(find . -name *.xml); do xmllint --format $f > /tmp/$$lint; cp /tmp/$$lint $f; done && rm /tmp/$$lint
```

Comment:
Thanks. I wish it wasn't so complicated, but I'm glad to know what to do

Reply:
Should  be fixed now.